enterprise/backend/src/metabase_enterprise/sandbox/models/params/field_values.clj

```diff
@@ -1,48 +1,14 @@
 (ns metabase-enterprise.sandbox.models.params.field-values
   (:require [metabase-enterprise.sandbox.api.table :as table]
             [metabase.api.common :as api]
-            [metabase.models.field :as field]
-            [metabase.models.field-values :as field-values :refer [FieldValues]]
+            [metabase.models.field :as field :refer [Field]]
+            [metabase.models.field-values :as field-values]
             [metabase.models.params.field-values :as params.field-values]
             [metabase.public-settings.premium-features :refer [defenterprise]]
             [toucan.db :as db]))
 
 (comment api/keep-me)
 
-(defn- create-sandboxed-field-values
```

Comment:
Removed and created a more generic function to create `advanced` FieldValues in `metabase.models.params.field-values`.